src/test/node/items/controllers.js

```diff
@@ -95,3 +95,52 @@ test('delete an item from database', t => {
     });
   });
 });
+
+test('delete all item from database', t => {
+  const ops = [
+    { type: 'put', key: 'item1', value: itemRedSelo },
+    { type: 'put', key: 'item2', value: itemAlaska }
+  ];
+
+  testDB.batch(ops, (err) => {
+    if (err) {
+      t.end(err);
+    }
+
+    const expected = {
+      status: 200,
+      message: 'success'
+    };
+
+    const req = httpMocks.createRequest({
+      method: 'DELETE',
+      url: '/items'
+    });
+
+    const res = httpMocks.createResponse();
+
+    ItemController.removeAll(req, res, () => {
+      const status = res.statusCode;
+      const message = res._getData().message;
+      testDB.get('item1', (err1) => {
+        if (err1 && err1.notFound) {
```

Comment:
there could be a database internal error. 
[https://github.com/Level/levelup#dbgetkey-options-callback](url)
Later we should talk about this issue.